user-guide/Standard_Apps/EPM/EPM_VSAT/VSAT_deployment.md

```diff
@@ -4,15 +4,117 @@ uid: VSAT_deployment
+
+## Collectors configuration
+
+### iDirect Evolution collector
+
+1. In DataMiner Cube, create an element using the *iDirect Evolution Platform VSAT* connector.
+
+   See [Adding elements](xref:Adding_elements).
+
+1. On the **Database Configuration** page of the new element, fill in the *Database Configuration* table.
+
+1. On the **DB Polling Config** page, adjust the polling parameters according to your preferences.
+
+1. On the **Collector Setup** page, configure the following settings:
+
+   - **File Import**: Enable *File Import* and *File Export* if CMDB (customer data) is available.
+   - **ID Import/Export**: Enable this feature and specify the paths to the CA and CR folder.
+   - **System Credentials**: Enter the system credentials.
+   - **Entity Removal File Path**: Specify the path for the entity removal file.
+
+1. Right-click the element and add the *NETWORK* custom property.
+
+   See [Adding a custom property](xref:Managing_element_properties#adding-a-custom-property-to-an-item).
+
+### Newtec Dialog collectors
+
+1. In DataMiner Cube, create the collector elements using the *Newtec Dialog Platform VSAT* connector.
+
+   When configuring the element connections, use the same HTTPS URL for all three required HTTP connections.
+
+1. On the **User Authentication** page of the new elements, enter the usernames and passwords.
+
+1. On the **Polling Config** page, set *TSDB Polling Status* to enabled, adjust the TSDB parameters as needed, and click *Auto Configure*.
+
+1. On the **Polling Status** page, adjust the polling parameters according to your preferences.
+
+1. On the **Collector Setup** page, configure the following settings:
+
+   - **File Import**: Enable *File Import* and *File Export* if CMDB (customer data) is available.
+   - **ID Import/Export**: Enable this feature and specify the paths to the CA and CR folder.
+   - **System Credentials**: Enter the system credentials.
+   - **Entity Removal File Path**: Specify the path for the entity removal file.
+
+1. Right-click the element and add the *NETWORK* custom property.
+
+## EPM configuration
+
+Follow the steps below to set up the **front-end and back-end elements**:
+
+1. In DataMiner Cube, create the front-end and back-end elements using the *Skyline EPM Platform VSAT GEO* connector.
+
+   Make sure you carefully plan your elements across the DMS. For more info, see [EPM VSAT architecture](xref:VSAT_architecture). You will configure these elements in a later step.
+
+1. For each of the collectors, the front end, and the back-end elements, right-click the element, select *Properties*, and take note of the element ID (in the format "DMAID/ElementID") displayed in the *Properties* window.
+
+   You will need these IDs in the next steps to configure the EPM elements.
+
+1. Configure the **front-end element**, using the Visual page of the element:
```

Comment:
Before your last commit, the front-end config also included the system credentials, file handling, and backup settings. Was this a mistake, or should this still be added here as well?